### DISCUSSION TITLE
ScM%20CS%40Brown/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#69] [Announcements]
# ScM%20CS%40Brown/

https://csmsapp.github.io/ScM%20CS%40Brown/

**Comment by beckysx** [comment]
美本3.7 无实习 2科研 录取了， 最后选择uci mcs，brown很好就是学费贵了一些，每个学期只需要两门课，考虑回国可以选这个

**Comment by drmeerkat** [comment]
这个必须评论一下，布朗对转博非常的友好，教授人都很好，课程压力不那么大可以有很多时间做科研。另外因为项目小，学校对待ms就像本科生一样，真的很亲切，paid RA也很好拿，机会非常多。个人觉得可以媲美S以上的项目如果你想申博。
我知道的数据点就有每年想申请博士的都基本上岸了，要么留本校要么去top30，top10还是有点难的需要个人更多努力（这个在哪都一样），但top30就很稳了。

**Reply by Ericlalalll** [reply]
老哥,可不可以再推荐些转博友好的项目呢, S级以下,谢谢

**Comment by Yangxinyee** [comment]
申请者的专业名不是 CS 就别送钱了，是真的嘛？本科专业名字是人工智能，课程内容算是计算机加上自动化，这种可以试试嘛？

**Reply by xichenpan** [reply]
我觉得肯定不会，刚帮忙看了下datapoints里面就有se的同学录了，但是overall不是cs本的同学好像就他一个录了，其他的都拒了，brown还是比较prefer科班的

**Reply by charlesyangxinye** [reply]
哦哦好的 谢谢哥！

**Reply by Ericlalalll** [reply]
请问本科是数学与计算机,学过编程语言,离散,算法,数据结构还有一些高阶算法和计算理论的课,请问可以被当作科班嘛

**Reply by Fridge003** [reply]
可以录，我的本科专业是智能科学与技术，上周拿到了brown scmcs的AD。但我修了很多cs的系统课，比如os，网络这些，可能有关系

**Comment by TadejPogacar1** [comment]
本科专业是information engineering，申这个是不是毫无可能

**Reply by xichenpan** [reply]
datapoints里应该有非CS录取的，可以看下，但好像总体看起来比较rare

**Comment by Fridge003** [comment]
感觉低了，可以提到A的

**Reply by Immortal-Dream** [reply]
感觉确实可以提

**Comment by wenquanlu** [comment]
24 fall offer群内调查结果：https://docs.google.com/presentation/d/11ruKcNZc-_Y7Uu4o1E9Vw-oBi05lb4Gy/edit?usp=sharing&ouid=108807744109224173527&rtpof=true&sd=true

**Reply by wenquanlu** [reply]
注：调查是自愿形式非强制

**Reply by JadeXu16** [reply]
好像第七页缺失了，是本科专业吗，非科班的很想要这个参考

**Comment by SimonXiangD** [comment]
本科数学，没有cs minor但是上了一些cs核心课，也录了

**Reply by SimonXiangD** [reply]
补一下，是陆本

**Reply by JadeXu16** [reply]
可以请问一下上了哪些核心课吗

**Reply by SingleView11** [reply]
程序设计，数据结构，操作系统，深度学习

**Reply by JadeXu16** [reply]
谢谢～！

**Comment by DemonRain7** [comment]
t 104不想再考了有机会么，GRE329+4

**Comment by Harry-Junhua-Huang** [comment]
美本t50cs gpa3.85,1实习两科研(水刊二座和再投3作）ddl投递2025. 3.21录取了

**Reply by Harry-Junhua-Huang** [reply]
看到地里面有大佬录取+25%奖学金了，我没奖hhh

**Comment by onlyivvvyli** [comment]
25fall 陆本985软件工程录了

**Comment by Llawlietcyd** [comment]
美本top50cs gpa 3.8 2科研录了
